docs/sql-ref-datatypes.md

```diff
@@ -631,4 +631,71 @@ from pyspark.sql.types import *
 </tr>
 </table>
 </div>
+
+<div data-lang="sql"  markdown="1">
+
+The following table shows the type names as well as aliases used in Spark SQL parser for each data type.
+
+<table class="table">
+<tr>
+  <th style="width:50%">Data type</th>
+  <th>SQL name</th></tr>
+<tr>
+  <td> <b>BooleanType</b> </td>
+  <td> boolean </td>
+</tr>
+<tr>
+  <td> <b>ByteType</b> </td>
+  <td> byte, tinyint </td>
+</tr>
+<tr>
+  <td> <b>ShortType</b> </td>
+  <td> shot, smallint </td>
+</tr>
+<tr>
+  <td> <b>IntegerType</b> </td>
+  <td> int, integer </td>
+</tr>
+<tr>
+  <td> <b>LongType</b> </td>
+  <td> long, bigint </td>
+</tr>
+<tr>
+  <td> <b>FloatType</b> </td>
+  <td> float, real </td>
+</tr>
+<tr>
+  <td> <b>DoubleType</b> </td>
+  <td> double </td>
+</tr>
+<tr>
+  <td> <b>DateType</b> </td>
+  <td> date </td>
+</tr>
+<tr>
+  <td> <b>TimestampType</b> </td>
+  <td> timestamp </td>
+</tr>
+<tr>
+  <td> <b>StringType</b> </td>
+  <td> string </td>
+</tr>
+<tr>
+  <td> <b>BinaryType</b> </td>
+  <td> string </td>
+</tr>
+<tr>
+  <td> <b>BinaryType</b> </td>
+  <td> string </td>
+</tr>
+<tr>
+  <td> <b>DecimalType</b> </td>
+  <td> decimal, desc, numeric </td>
```

Comment:
Ur, `desc`?

Reply:
Sorry for the rush, should not push before dinner... Done in 99d0db6.